velox/exec/HashProbe.cpp

```diff
@@ -59,14 +59,15 @@ void extractColumns(
     folly::Range<char**> rows,
     folly::Range<const IdentityProjection*> projections,
     memory::MemoryPool* pool,
-    const RowVectorPtr& result) {
+    const std::vector<TypePtr>& resultTypes,
+    std::vector<VectorPtr>& resultVectors) {
   for (auto projection : projections) {
```

Comment:
nit: Should we add a dcheck to ensure resultVectors/types size > number of channels.